data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/GenericExpressionEvaluator_ConditionalIT.java

```diff
@@ -192,6 +192,8 @@ private static Stream<Arguments> validExpressionArguments() {
                 arguments("/should_drop", event("{\"should_drop\": true}"), true),
                 arguments("/should_drop", event("{\"should_drop\": false}"), false),
                 arguments("/logs/2/should_drop", event("{\"logs\": [{}, {}, {\"should_drop\": true}]}"), true),
+                arguments("/path == " + rawString("/path/to/route"), event("{\"path\": \"/path/to/route\"}"), true),
```

Comment:
This code would be cleaner just having the whole syntax here. That is, we don't need a `rawString` method.

Reply:
Removed this helper method in favor of inline.